nsv13/code/modules/munitions/ship_weapons/hybrid_weapons/hybrid_railgun.dm

```diff
@@ -138,7 +138,7 @@
 	var/obj/item/ship_weapon/ammunition/T = chambered
 	if(T)
 		var/final_velo = projectile_velo - ((100 - alignment) / 100) //Misalignment slows projectiles
-		linked.fire_projectile(T.projectile_type, target, FALSE, final_velo, null, TRUE)
+		linked.fire_projectile(T.projectile_type, target, FALSE, final_velo, TRUE, TRUE) //CHECK THIS CODE LATERAL WAS RECEIVING NULL, REPLACED TO TRUE
```

Comment:
I request scrutiny on this, It fixed the issue but i want confirmation that it looks good.

Reply:
Previously this was irrelevant and overridden, but I guess it doesn't do that anymore. Shrug.

Reply:
So looking at the definition.... 
https://github.com/BeeStation/NSV13/blob/617bfd3c4a58b218f4b362aa92624f7af08f6af7/nsv13/code/modules/overmap/physics.dm#L592
/obj/structure/overmap/proc/fire_projectile(proj_type, atom/target, speed=null, user_override=null, lateral=FALSE, ai_aim = FALSE, miss_chance=5, max_miss_distance=5, broadside=FALSE)

the third argument is speed, which says... FALSE? fourth is user_override which is set to the velocity, fifth is lateral as you identified, sixth is the aimbot which is true...? on second thought, let's just use the names

```suggestion
		linked.fire_projectile(T.projectile_type, target, speed=final_velo, user_override=TRUE, lateral=TRUE) //CHECK THIS CODE LATERAL WAS RECEIVING NULL, REPLACED TO TRUE
```

Reply:
This accepted edit has fixed the guns even more, they now are firing the projectile faster, Neat